portfolio/src/main/webapp/script.js

```diff
@@ -27,11 +27,16 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function fetchName() {
+function getComments() {
   const messageContainer = document.getElementById('comments')
-  fetch('/data').then(response => response.json()).then(name => name.reduce((acc, curVal) =>
-    acc + "<p>" + curVal + "</p></br>")
-  ).then( (comments) => {
-    messageContainer.innerHTML = comments;
-  } );
+  fetch('/data').then( response => response.json()).then(comments => comments.reduce((acc, curVal) =>  {
+      return acc + "<p>" + curVal.comment + " - <b>" +  curVal.userName +"</b></p></br>"
+    }, "")
+    ).then( (comments) => messageContainer.innerHTML = comments);
```

Comment:
Nit: define your lambda consistently. I see `.then( response =>`, `.then(comments =>` and `.then( (comments) =>`. I happen to personally prefer the second one but any consistent style is a good style.

Reply:
done!